demos/can-view-callbacks/dynamic_tooltip.html

```diff
@@ -24,13 +24,13 @@
 		<tr>
 			<td colspan="3">
 				<button tooltip="{{deleteTooltip()}}"
-						{{^selected.length}}class="disabled"{{/selected.length}}
+						{{^if(selected.length)}}class="disabled"{{/if}}
```

Comment:
did `{{^selected.length}}` not work?

Reply:
I think it works but generates warnings if `selected` is `undefined`, so I thought this would be a better way of writing the same thing.

Reply:
Ok, we've got an issue open to fix that. Thanks.